experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -321,17 +321,6 @@ export class FetchInstrumentation extends InstrumentationBase<Promise<Response>>
 
         function endSpanOnSuccess(span: api.Span, response: Response) {
           plugin._applyAttributesAfterFetch(span, options, response);
-          const spanResponse = {
-            status: response.status,
-            statusText: response.statusText,
-            headers: response.headers,
-            url
-          };
-          if (response.status >= 200 && response.status < 400) {
-            if (response.url != null && response.url !== '') {
-              spanResponse.url = url;
-            }
-          }
           plugin._endSpan(span, spanData, {
             status: response.status,
             statusText: response.statusText,
```

Comment:
now `headers` might be missing, right? :thinking: 
the original code of this method was:
```
        function endSpanOnSuccess(span: api.Span, response: Response) {
          plugin._applyAttributesAfterFetch(span, options, response);
          if (response.status >= 200 && response.status < 400) {
            plugin._endSpan(span, spanData, response);
          } else {
            plugin._endSpan(span, spanData, {
              status: response.status,
              statusText: response.statusText,
              url,
            });
          }
        }
```

Reply:
Original code it is: https://github.com/open-telemetry/opentelemetry-js/pull/3037/commits/df2c8a9434b6e2d81e690c773cc6054e0d2009b8